src/components/Instructors.jsx

```diff
@@ -0,0 +1,122 @@
+import { useRef, useState } from "react";
+import styled from "styled-components";
+
+import { instructors } from "../libraries/instructors.json";
+import { InstructorsSliderCard } from "./InstructorsSliderCard";
+
+export const Instructors = () => {
+  const [scrollPosition, setScrollPosition] = useState(0);
+
+  const containerRef = useRef();
+
+  const handleScroll = (scrollAmount) => {
```

Comment:
We think it'd be better if this handle scroll function can be extracted and put into a component so it can be reused in several components.

//Sofie & Wen